app/config/mq-config.js

```diff
@@ -12,6 +12,12 @@ const mqSchema = joi.object({
     address: joi.string().default('payment'),
     username: joi.string(),
     password: joi.string()
+  },
+  eventTopic: {
+    name: joi.string().default('ffc-pay-event'),
```

Comment:
Doesn't need a name.  I'd remove it to simplify unless the npm package needs it.

Reply:
removed